src/ansible/roles/keycloak/tasks/main.yml

```diff
@@ -0,0 +1,148 @@
+- name: print keycloak_url
+  debug:
+    msg: "keycloak_url={{ keycloak_url }}"
+
+- name: Download software zipfile
+  get_url:
+    url: "{{ keycloak_url }}"
+    dest: /tmp
+    
+- name: Unzip software
+  unarchive:
+    remote_src: yes
+    src: "/tmp/{{ zip_file }}"
+    dest: /tmp
```

Comment:
I believe there is a way to omit the root folder from the zip file and set custom name, so you can extract it to /tmp/keycloak instead of a versioned folder.